### PR TITLE
draft release notes for dcos 2.0.5

### PR DESCRIPTION
**Pull request by cmastr**
## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.

https://jira.d2iq.com/browse/D2IQ-69667

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [x] Create your PR against `staging`, not `master`. 
- [ ] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/mesosphere/dcos/2.0/release-notes/2.0.5/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+- Apache&reg; Mesos&reg; 1.9.1-dev
+- OpenSSL 1.1.1g
+
+### DC/OS Fixed and Improved Issues
+
+- Fixed a critical error in Metronome where existing jobs appear to be lost after upgrade. (COPS-6174)
+- Fixed an issue with unmounting external persistent volumes in Mesos. (COPS-5920) 
+
+# Marathon Fixed and Improved Issues
```
Comment (cneth) [inline pages/mesosphere/dcos/2.0/release-notes/2.0.5/index.md:33]: I don't see any bump to Marathon in this release.  Did we change Marathon?

File: pages/mesosphere/dcos/2.0/release-notes/2.0.5/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+# DC/OS 
+
+## Components
+
+DC/OS 2.0.5 includes the following component versions:
+
+- Apache&reg; Mesos&reg; 1.9.1-dev
+- OpenSSL 1.1.1g
+
+### DC/OS Fixed and Improved Issues
```
Comment (cneth) [inline pages/mesosphere/dcos/2.0/release-notes/2.0.5/index.md:28]: Missing items:
1. D2IQ-68394 COPS-6128: "Fixed a problem where Zookeeper log messages were not being forwarded to syslog."
2. (No JIRA), but the PR is here: https://github.com/dcos/dcos/pull/7328
   Suggested wording: "Fixed a problem where the DC/OS UI sometimes did not report the correct DC/OS version due to browser caching".
3. (D2IQ-68800) PR is here: https://github.com/dcos/dcos/pull/7341  
   Suggested wording: "Fixed a problem where frameworks could interfere with Marathon pods by launching tasks on resources reserved to Marathon."

Comment (cmastr) [reply pages/mesosphere/dcos/2.0/release-notes/2.0.5/index.md:28]: @cneth see below:
1. added to release notes but used previously written description (2.1 release notes)
2. Waiting on development to add a JIRA before I can add this. 
3. This item is not a COPS ticket, and @vinodkone has instructed to only add COPS tickets for DCOS at this time. Will need to wait for clarification from him.

Comment (vinodkone) [reply pages/mesosphere/dcos/2.0/release-notes/2.0.5/index.md:28]: @cneth The only COPS tickets recommendation was for minor releases only. For patch releases, I think we should include all relevant JIRAs.

Comment (vinodkone) [reply pages/mesosphere/dcos/2.0/release-notes/2.0.5/index.md:28]: created https://jira.d2iq.com/browse/D2IQ-69682 for 2.

File: pages/mesosphere/dcos/2.0/release-notes/2.0.5/index.md (hunk 1 above)
Comment (cmastr) [reply pages/mesosphere/dcos/2.0/release-notes/2.0.5/index.md:35]: no change to marathon.

File: pages/mesosphere/dcos/2.0/release-notes/2.0.5/index.md (hunk 2 above)
Comment (cmastr) [reply pages/mesosphere/dcos/2.0/release-notes/2.0.5/index.md:28]: Ok, items added.

**Pull request merged**